go.mod

```diff
@@ -4,7 +4,8 @@ go 1.16
 
 require (
 	github.com/gofiber/fiber v1.14.6
+	github.com/gofiber/fiber/v2 v2.18.0
 	github.com/mattn/go-sqlite3 v1.14.8 // indirect
-	gorm.io/driver/sqlite v1.1.4 // indirect
-	gorm.io/gorm v1.21.13 // indirect
+	gorm.io/driver/sqlite v1.1.4
```

Comment:
I'd probably document, just to be on the safe side, that SQLite will work great for very small testing/tutorial environments, but to use a full-scale RDBMS like MySQL/PostgreSQL/etc if they're building something more production-ready.